### PR TITLE
Added an option to validate all SNPs

### PR DESCRIPTION
**Pull request by nevrome**
Just as suggested in #223.

### CODE REVIEW

**Review by stschiff — APPROVED** [review]
Looks good. Why is test/testDat/poseidonHSGoldenTestData/validate3 empty?

**Comment by stschiff** [comment]
I'll try and quickly implement a file size check now.

File: src/Poseidon/CLI/OptparseApplicativeParsers.hs [hunk 1]
```diff
@@ -391,6 +391,13 @@ parseIgnoreGeno = OP.switch (
     OP.hidden
     )
 
+parseFullGeno  :: OP.Parser Bool
+parseFullGeno = OP.switch (
+    OP.long "fullGeno" <>
+    OP.help "test parsing of all SNPs" <>
```
[the remark this answers is not in the document]
Comment (nevrome) [reply src/Poseidon/CLI/OptparseApplicativeParsers.hs:397]: Good point!

**Comment by nevrome** [comment]
`test/testDat/poseidonHSGoldenTestData/validate3` is empty, because `validate` writes nothing to stdout. It just writes to stderr (logging) and then returns an exit code. Call it lazy programming, but I kept the empty file around to detect any change to this behaviour :grimacing:. Same holds for `validate1` and `validate2`. Would you prefer to test this more directly?

**Comment by stschiff** [comment]
No, that's OK... indeed the Golden tests are lazy but robust. All good.

**Pull request merged**